setup.php

```diff
@@ -92,6 +92,30 @@ public function enqueue_scripts($page_hook) {
     );
     wp_enqueue_script('yoast-cmb2-plugin-js');
   }
+
+
+  /**
+   * Test whether Yoast SEO is hidden either by choice of the admin or because
+   * the post type is not a public post type
+   *
+   *
+   * @param  string $post_type (optional) The post type to test, defaults to the current post post_type.
+   *
+   * @return  bool        Whether or not the yoast seo metabox should be hidden
+   */
+  function check_if_yoast_seo_is_hidden( $post_type = null ) {
+    if ( ! isset( $post_type ) && ( isset( $GLOBALS['post'] ) && ( is_object( $GLOBALS['post'] ) && isset( $GLOBALS['post']->post_type ) ) ) ) {
```

Comment:
same as before, if we could remove any unnecessary whitespace (I appreciated this has just been lifted from the the plugin code). If we could do the same for lines 111 onwards too?

Perhaps its also worth storing `$GOLBALS` as a new variable here before the `if` conditions so that we aren't repeatedly checking agains the globals variable directly?

Could we also look to break this long line up too? I tend to try and aim for around 80 characters per line, a line return and 1 indent usually still looks readable